src/domain/datasets/services/src/dependencies.rs

```diff
@@ -45,6 +45,8 @@ pub fn register_dependencies(b: &mut dill::CatalogBuilder, needs_indexing: bool)
     b.add::<DatasetReferenceServiceImpl>();
     b.add::<DatasetStatisticsServiceImpl>();
     b.add::<DatasetIncrementQueryServiceImpl>();
+    b.add::<AccountQuotaCheckerStorageImpl>();
+    b.add::<kamu_core::DidSecretService>();
```

Comment:
It is highly questionable that we register `kamu_core::DidSecretService` within `kamu-datasets-services`, which is from another crate